### ISSUE TITLE
Using LocalStorage in main.dev.js will not trigger 'window-all-closed' and after app quit, 4 dangling processes will remain

### ISSUE THREAD

**Issue by jalanga** [jerry1100/electron-browser-storage#4]
Electron: 7.1.2
electron-browser-storage: 1.0.6

In main.dev.js on app ready, I init my session:

```
function initSession(){
  // Session Config
  const filter = {
    urls: [constants.webRequestUrlFilter + '*']
  };

  session.defaultSession.webRequest.onBeforeSendHeaders(filter, (details, callback) => {
    details.requestHeaders.Origin = constants.originHeader;

    localStorage.getItem('AUTH:authToken').then(function(authToken){
      if (authToken){
        details.requestHeaders['AUTH-TOKEN'] = authToken;
      }
      callback({
        cancel: false,
        requestHeaders: details.requestHeaders
      });
    });


  });
}
```

Just the import of LocalStorage `const { localStorage } = require('electron-browser-storage');
`, will make 4 dangling processes after app quits.

Also `window-all-closed` is not called whit localStorage is imported

**Comment by jerry1100** [comment]
Thanks for reporting. While I look into this, what platform are you running on? Windows, Mac, Linux?

**Comment by jerry1100** [comment]
Hey @jalanga, I just published v1.0.7 which fixes this issue. Please give it a try.
